README.md

```diff
@@ -9,7 +9,7 @@ with [FrankenPHP](https://frankenphp.dev) and [Caddy](https://caddyserver.com/)
 
 1. If not already done, [install Docker Compose](https://docs.docker.com/compose/install/) (v2.10+)
 2. Run `docker compose build --no-cache` to build fresh images
-3. Run `docker compose up --pull always -d --wait` to start the project
+3. Run `docker compose up --pull always -d --wait` to set up and start a fresh Symfony project
```

Comment:
```suggestion
3. Run `docker compose up --pull always -d --wait` to start the project (or create a new one if no project is found)
```

Reply:
I think that this suggestion is confusing, there is a dedicated involving another process for existing apps.

Reply:
I think it's viable in the getting started block to say it creates a new symfomy project

Reply:
As these are sufficient, it is already indicated in the description and the first lines of the README that it is an “installer and runtime”.
Otherwise, if you strictly follow the "getting started" instructions, you can see that an installation has been made.